src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -47,27 +55,40 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
-	public String changeTheFirstLetterToUppercase(String word) {
-
-		String[] regexResults = word.split("_");
+	public void getDataFromFile() {
 
+		File file = new File("D:/java/europe_capitals.txt");
 		StringBuilder stringBuilder = new StringBuilder();
+		String dataFromFile = null;
```

Comment:
Tej zmiennej nie używasz nigdzie poza końcowym fragmentem metody, więc nie ma potrzeby jej inicjalizować tutaj, a można by i argumentować że nie ma potrzeby jej tworzyć w ogóle.

Reply:
OK, postaram się to poprawić.

Reply:
Poprawione. Usunąłem w ogóle tę zmienną.